src/dspeed/processors/multi_a_filter.py

```diff
@@ -1,3 +1,5 @@
+"""Processors implementing multi-A trapezoidal filters."""
```

Comment:
Change to:
Processors for identifying multiple maxima and their amplitudes in a waveform